docs(mac): Document What's New for version 17.0 (#10764)
keymanapp/keyman: mac/help/about/whatsnew.md

Here are some of the new features we have added to Keyman 17.0 for macOS:

> Keyman Engine for macOS depends on Keyman Core for processing keystrokes, improving reliability and consistency with the Linux and Windows versions of Keyman (#10066)
> 
> Move active keyboards list to Keyman section of main Input Menu from hierarchical submenu (#9777)

Reviewer: ```suggestion
* Keyman Engine for macOS now uses Keyman Core for processing keystrokes, improving reliability and consistency with the Linux and Windows versions of Keyman (#10066)
* List of Keyman keyboards has been moved out of a submenu of the main Input Menu into the Keyman section of the menu (#9777)
```
